README.md

```diff
@@ -160,7 +160,7 @@ Used to maintain the code style for the whole project.
 * 0.0.1 first prototype using Hadoop MapReduce and Apache Giraph for operator
  processing
 * 0.0.2 support for HBase as distributed graph storage
-* 0.0.3 Apache Flink replaces MapReduce and Giraph as operator implementation
+* 0.0.3 Apache Flink replaces MapReduce and Giraph as operator iterationStrategy
```

Comment:
fix your search & replace approach (btw, you can limit this to source files and exclude docs)